dashboards-observability/public/components/event_analytics/explorer/explorer.tsx

```diff
@@ -1073,6 +1075,7 @@ export const Explorer = ({
         isIndexPatternChanged(tempQuery, query[RAW_QUERY])
```

Comment:
What is index pattern? When does it change?

Reply:
it's the index in `source = <index>`, user changes it manually. If user changes it the pattern field should reset